documentation/docs/08-events.md

```diff
@@ -2,10 +2,6 @@
 title: Events
 ---
 
-SvelteKit emits [CustomEvents](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent) on the `window` object when certain things happen:
+SvelteKit emits a `sveltekit:start` [CustomEvents](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent) on the `window` object once the app has hydrated.
```

Comment:
would be good to do client-side hooks so that we can get rid of this last custom event